Assets/Scripts/Entities/Movement.cs

```diff
@@ -24,6 +24,12 @@ public void Start()
         private float _angle;
         public void Update()
         {
+            if (Target == null)
+            {
+                this.GetComponent<BehaviorExecutor>().enabled = false;
+                this.enabled = false;
+                return;
```

Comment:
zou je geen target moeten zetten in plaats van niks doen? Nu doet een enemy niks als hij wordt geïnstantieerd

Reply:
dat klopt. dit is voor het neerzetten voordat de wedstrijd begint. Als de wedstrijd begint wordt de target gezet zodat hij dan begint met aan te vallen. Het is ook niet zeker als dit geinstantieerd is dat er wel een player al is.
In mijn andere scriptje de battle manager daar wordt een npc geinstantieerd met target